### PR TITLE
accept multi-byte comment when run code

### PR DESCRIPTION
**Pull request by mogimogitomato**
Add utf-8 encoding when sending code to sonic-pi.

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -53,7 +54,8 @@ class SonicPi {
         if(!editor) {
             return;
         }
-        return editor.document.getText();
+        const iconv = require('iconv-lite');
```
Comment (mactkg) [inline src/extension.ts]: I think that we can import `iconv-lite` at top of this file.  Do you have any reason to require the library here?

File: src/type.d.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+declare module 'osc-js';
+declare module 'iconv-lite';
```
Comment (mactkg) [inline src/type.d.ts]: I found `index.d.ts` file for `iconv-lite`: https://github.com/ashtuchkin/iconv-lite/blob/17d00ab8cd663acafe390d8224ca1d44cb0b2085/lib/index.d.ts
Should we declare `iconv-lite`?

**Review by mactkg — COMMENTED** [review]
Thank you for your contribution!
I reviewed your code.  Please reply my comments.

File: src/extension.ts (hunk 1 above)
Comment (mogimogitomato) [reply src/extension.ts]: Sorry, I should modify `getCurrentCode()` type definition String to Buffer.  
but I tried to tsc without modify, called require, not import.  
It's wrong, so I fixed mistakes.

File: src/type.d.ts (hunk 2 above)
Comment (mogimogitomato) [reply src/type.d.ts]: You're right, there're no need to declare `iconv-lite`.  
I restore modification.

**Review by mactkg — APPROVED** [review]
Approved. Thanks!

**Pull request merged**